lib/yabeda/schked.rb

```diff
@@ -41,6 +41,8 @@ def self.job_name(job)
       labels = {success: !job.opts[:failed], name: job_name(job)}
       Yabeda.schked.job_execution_runtime.measure(labels, job.last_work_time.round(3))
       Yabeda.schked.jobs_executed_total.increment(labels)
+
+      job.opts[:failed] = false
```

Comment:
Probably good idea to place it under `ensure` for two reasons:
 1. to reset flag even in case if something errored in Yabeda itself
 2. to visually make this code as a cleanup code
```suggestion
    ensure
      job.opts[:failed] = false
```